site/src/examples/navigation-item/RenderElement.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useState } from "react";
+import {
+  NavigationItem,
+  NavigationItemRenderProps,
+  StackLayout,
+  Text,
+} from "@salt-ds/core";
+
+export const RenderElement = (): ReactElement => {
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const CustomLinkImplementation = (props: NavigationItemRenderProps) => (
```

Comment:
We should move this out of the component

Reply:
done